Library/Formula/slib.rb

```diff
@@ -0,0 +1,24 @@
+class Slib < Formula
+  homepage "http://people.csail.mit.edu/jaffer/SLIB"
+  url "http://groups.csail.mit.edu/mac/ftpdir/scm/slib-3b4.zip"
+  version "3b4"
```

Comment:
SLIB version consists from 3 letters, major + minor + micro. and minor start from "a", then "b", then "c", and so on. I believe that this format is selected to make archive file name fit in 8+3 letters (for MS-DOS).

http://en.wikipedia.org/wiki/SLIB also says that 3b4 is stable release.